### PR TITLE
implements the updating of loans transactions simulateneously

### LINKED ISSUE #1261: New accounts are not added instantaneously
**Summary:**

New Saving/Loan accounts are not updated in the recyclerview after they are added

**Steps to reproduce:**

* Go to Accounts Fragment
* Click on add floating action button for Saving/Loan account
* Complete the adding procedure
* Observe that new account hasn't been added yet
* Swipe to refresh and now the new account is visible

**Expected behaviour:**

The new account should have been added in the recyclerview as soon it was made.

**Observed behaviour:**

The new account is added only after the user swipes to refresh

**Device and Android version:**

Xiaomi PocoF1
Android 9
Stock rom

**Comments:**

Good first issue for beginners

### PR DESCRIPTION
**Pull request by falayy**
Fixes #1261 

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/org/mifos/mobile/ui/fragments/base/BaseFragment.java [hunk 1]
```diff
@@ -73,6 +74,15 @@ public void onAttach(Context context) {
         }
     }
 
+    public void replaceFragment(Fragment fragment) {
```
Comment (devansh-299) [inline app/src/main/java/org/mifos/mobile/ui/fragments/base/BaseFragment.java]: You can the replaceFragment function of BaseActivity. This separate function is not needed.

File: app/src/main/java/org/mifos/mobile/ui/fragments/ClientAccountsFragment.java [hunk 2]
```diff
@@ -139,13 +139,12 @@ private void setUpViewPagerAndTabLayout() {
...
             @Override
             public void onPageScrolled(int position, float positionOffset,
-                    int positionOffsetPixels) {
+                                       int positionOffsetPixels) {
             }
 
             @Override
             public void onPageSelected(int position) {
                 getActivity().invalidateOptionsMenu();
-                ((HomeActivity) getActivity()).hideKeyboard(getView());
```
Comment (devansh-299) [inline app/src/main/java/org/mifos/mobile/ui/fragments/ClientAccountsFragment.java]: Removing this line is unnecessary and also add screen recording for the changes done :)

**Comment by laxyapahuja** [comment]
Please squash your commits!

**Comment by laxyapahuja** [comment]
I can still see 3 commits.

**Comment by falayy** [comment]
> I can still see 3 commits.

I am working on it, I have a bit of issue squashing my commits after creating a PR, so I am talking to my colleagues

**Comment by falayy** [comment]
ezgif com-video-to-gif

**Comment by devansh-299** [comment]
@corneliuze still your branch is 6 commits behind the openMF/mifos-mobile development branch.

You can follow these commands to fix this
* `git remote add upstream https://github.com/openMF/mifos-mobile.git`  (if upstream wasn't set before)
* `git fetch upstream`
* `git checkout Fixes#1261`
* `git reset --hard upstream/development` (note :- this will overwrite all changes you did on your branch locally. Better to note the changes you did)
* Do the changes that you proposed to do
* Do git add etc and force push the changes to origin

File: app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.java [hunk 3]
```diff
@@ -121,7 +122,7 @@ public void onCreate(Bundle savedInstanceState) {
 
     @Override
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
-            @Nullable Bundle savedInstanceState) {
+                             @Nullable Bundle savedInstanceState) {
```
Comment (devansh-299) [inline app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.java]: No need to do this

File: app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.java [hunk 4]
```diff
@@ -457,7 +465,7 @@ public void onItemLongPress(View childView, int position) {
     /**
      * This function opens up an activity only if the intent
      * is not null.
-     *
+     * <p>
```
Comment (devansh-299) [inline app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.java:468]: you can remove this, else everything seems fine to me

**Comment by luckyman20** [comment]
PR is not relevant now.

**Pull request closed without merging**